pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -379,6 +379,7 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 					},
 				}
 			}
+			out.VXLANPort = in.VXLANPort
```

Comment:
You shouldn't need to manually set this here; the `DefaultConvert` should handle it